nemo/lightning/pytorch/strategies.py

```diff
@@ -100,19 +123,19 @@ def __init__(
         cluster_environment=None,  # TODO: Add type-hint
         checkpoint_io=None,  # TODO: Add type-hint
         find_unused_parameters: bool = False,
-        ckpt_include_optimizer: bool = False,
+        ckpt_include_optimizer: bool = True,
         ddp: Union[DDPLiteral, DistributedDataParallelConfig] = "megatron",
         lazy_init: bool = False,
         pipeline_dtype: Optional[torch.dtype] = None,
-        save_ckpt_format='torch_dist',
-        ckpt_async_save=False,
-        ckpt_torch_dist_multiproc=None,  ## TODO(ashors): put elsewhere?
-        ckpt_assume_constant_structure=False,
-        ckpt_parallel_save=True,
-        ckpt_parallel_save_within_dp=False,
-        ckpt_parallel_load=False,
-        ckpt_parallel_save_optim=True,
-        ckpt_load_directly_on_device=True,
+        save_ckpt_format: str = 'torch_dist',
+        ckpt_async_save: bool = False,
+        ckpt_torch_dist_multiproc: int = None,  ## TODO(ashors): put elsewhere?
```

Comment:
Is there a plan to leave those args here or actually move elsewhere?
How will they be configured?

Reply:
I think we'll leave them here for now, because finding a cleaner solution is nontrivial. They can be configured when the user creates their strategy, e.g. 
```
strategy = nl.MegatronStrategy(
    ckpt_save_format='zarr',
    ckpt_parallel_save_optim=False,
    ckpt_async_save=False,
)
```

Reply:
is it configurable with a yaml config?